libs/src/services/creatorNode/index.js

```diff
@@ -194,14 +207,24 @@ class CreatorNode {
    * @param {string?} sourceFile
    */
   async uploadTrackMetadata (metadata, sourceFile) {
-    return this._makeRequest({
-      url: '/tracks/metadata',
-      method: 'post',
-      data: {
-        metadata,
-        sourceFile
-      }
-    }, true)
+    // this does the actual validation before sending to the creator node
```

Comment:
same as comment for `uploadCreatorContent`

Reply:
see comment above